src/Sylius/Behat/.gitignore

```diff
@@ -0,0 +1 @@
+/phpspec.yml
```

Comment:
We never prefix files with / here. Is this gitignore needed here? I guess you should call bin/phpspec from the root dir.